.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,17 +1,15 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
-
----
-
-**Describe the bug**
+title: ""
```

Comment:
The reformatting broke the templates actually, as far as i can see.

Reply:
is it okay right now?

Reply:
yup, but same in _feature_request_